src/app/datasrc/datasrc.component.ts

```diff
@@ -29,6 +29,8 @@ const targetIntTypeId = 'WaveDataTargetIntType';
 const autoStreamViewId = 'WaveDataAutoStreamView';
 const manualStreamViewId = 'WaveDataManualStreamView';
 
+const communityMemberRoleTypeId = 'f79a55da-7c76-4600-a809-0f62ca9971d9';
```

Comment:
this feels like it would be better to hard code the guid in the library and down below say something similar to
`r.RoleTypeId === this.sdsservice.roles.communityMemberRoleTypeId`
or however the property would need to be referenced.

Reply:
I think you're right, more consistent with other samples that way as well. See if it looks better now.